source/faq.rst

```diff
@@ -1,264 +1,219 @@
 Frequently Asked Questions
-================================================================================
+##########################
 
 .. contents::
    :local:
    :depth: 1
 
+
+*********************************
 How can I contact the developers?
-================================================================================
+*********************************
+
+The best place to discuss PMM with developers and other community members is the `community forum <https://www.percona.com/forums/questions-discussions/percona-monitoring-and-management>`_.
+
+To report a bug, visit the `PMM project in JIRA <https://jira.percona.com/projects/PMM>`_.
 
-The best place to discuss PMM with developers and other community members
-is the `community forum <https://www.percona.com/forums/questions-discussions/percona-monitoring-and-management>`_.
 
-If you would like to report a bug,
-use the `PMM project in JIRA <https://jira.percona.com/projects/PMM>`_.
 
 .. _sys-req:
 
+*************************************************
 What are the minimum system requirements for PMM?
-================================================================================
+*************************************************
 
-.. rubric:: |pmm-server|
+.. rubric:: PMM Server
 
 Any system which can run Docker version 1.12.6 or later.
 
-It needs roughly 1 GB of storage for each monitored database node
-with data retention set to one week.
+It needs roughly 1 GB of storage for each monitored database node with data retention set to one week.
+
+.. note::
+
+   By default, :ref:`retention <data-retention>` is set to 30 days for Metrics Monitor and for Query Analytics.  You can consider :ref:`disabling table statistics <performance-issues>` to decrease the Prometheus database size.
+
+The minimum memory requirement is 2 GB for one monitored database node.
 
 .. note::
 
-   By default, :ref:`retention <data-retention>` is set to 30 days for
-   Metrics Monitor and for Query Analytics.  Also consider
-   :ref:`disabling table statistics <performance-issues>`, which can
-   greatly decrease Prometheus database size.
+   The increase in memory usage is not proportional to the number of nodes.  For example, data from 20 nodes should be easily handled with 16 GB.
+
+.. rubric:: PMM Client
 
-Minimum memory is 2 GB for one monitored database node, but it is not
-linear when you add more nodes.  For example, data from 20 nodes
-should be easily handled with 16 GB.
+Any modern 64-bit Linux distribution. It is tested on the latest versions of Debian, Ubuntu, CentOS, and Red Hat Enterprise Linux.
 
-.. rubric:: |pmm-client|
+A minimum of 100 MB of storage is required for installing the PMM Client package.  With a good connection to PMM Server, additional storage is not required.  However, the client needs to store any collected data that it cannot dispatch immediately, so additional storage may be required if the connection is unstable or the throughput is low.
```

Comment:
You may want to be explicit that caching applies to Query Analytics data only, as Prometheus data is never cached on the client side.